src/scripts/scenes/WorldEditorScene.js

```diff
@@ -199,7 +200,17 @@ class WorldEditorScene extends Scene {
   }
   
   addCurrentGameObject(x, y) {
-    let currentGameObject = null;
+    var en = this.curEntity;
```

Comment:
Change `en` to `entity`, and `go` to `gameObject`. `img` can be `img` or `image`, either is ok by me.